app/atoms/ln-patternlab/_ln-m-widget-contact.scss

```diff
@@ -0,0 +1,26 @@
+$_phone_prefix: 'T';
```

Comment:
All atoms would be in patternlab, plus lots of molecules. Are you proposing something like this?

```
app/
  patternlab/
    atoms/
    molecules/
  custom/
    atoms/
    molecules/
````

Where overrides to patternlab componenets go in the patternlabs folder, and custom compoenents in the custom folder.